.gitignore

```diff
@@ -2,45 +2,40 @@ syntax: glob
 
 ### VisualStudio ###
 
-# Tool Runtime Dir
+# Tools directory
+/[Tt]ools/
 .dotnet/
 .packages/
-.tools/
 
 # User-specific files
 *.suo
 *.user
 *.userosscache
 *.sln.docstates
+launchSettings.json
 
 # Build results
+
 artifacts/
-.idea/
+artifacts_stage_1/
 [Dd]ebug/
-[Dd]ebugPublic/
 [Rr]elease/
-[Rr]eleases/
-build/
-bld/
+x64/
+x86/ !eng/common/cross/x86/
```

Comment:
Is the space here correct?

Reply:
Same here, I copied the file from Arcade. I can use the runtime one here as well.

Reply:
yes, the space here is correct.